src/arm/bbb-bone-buses.dtsi

```diff
@@ -485,7 +485,12 @@ bone_timer_5: &not_available {
 
 // McASP
 bone_mcasp_0: &mcasp0 {
-
+        serial-dir = <  /* 0: INACTIVE, 1: TX, 2: RX */
+		2 0 1 0
```

Comment:
Interesting that this is different. Is it possible to instead put the I2S (I think this is I2S) in as a peripheral defined as part of the main board (bone bus) device tree and simply add target device specific (not bus pinout specific) settings in this file?

Reply:
Oh, nevermind. That is exactly what you are doing. I completely lost the fact that bbb-bone-buses and bbai-bone-buses *ARE* the bone bus definitions and *NOT* the overlays. My bad on silly comments.